### PR TITLE
Add view tests

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: tests/views/tp.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
```
Comment (ta2-1) [inline tests/views/tp.py]: We are using `translation_project` instead of `tp` in module names. As I understood a short version was intended here.

**Comment by ta2-1** [comment]
@phlax, I was going to say that tests are great and it's not worth to check everything and the main thing is that tests pass... But they don't now.

**Comment by phlax** [comment]
> @phlax, I was going to say that tests are great and it's not worth to check everything and the main thing is that tests pass... But they don't now.

sure i was trying to fix the calculate_checks on langs issue that only happens on travis

i can remove it if you are gtm re the code - all else passes reliably afaict

**Comment by ta2-1** [comment]
lgtm, since tests will pass again

**Pull request merged**